shim/README.md

```diff
@@ -0,0 +1,12 @@
+# `cc-shim`
+
+`cc-shim` is a process spawned by the runtime per container.The runtime 
+provides the pid of the cc-shim process to containerd-shim on OCI create command.
+
+`cc-shim` forwards all signals to the cc-proxy process to be handled by the agent
+in the VM.
+
+TODO:
+It should also forward any input received from containerd-shim to cc-proxy and 
+forward the stdout/stderr received from the proxy to containerd-shim.
```

Comment:
@dlespiau what is the plan for tty/stdio manage ?
cc-oci-runtime have 3 stdio cases to handle
if ( not config->console ) // not using --console something
        if (detach_mode)
                send io to docker named pipes //send to pipes used by "docker logs container_id"
        else if( attach_mode ) // running standalone no console 
                send io to console.socket 

else have config->console // using --console something
        use the console provided to show stdio